sdk/containerregistry/azure-containerregistry/azure/containerregistry/_container_registry_client.py

```diff
@@ -451,13 +451,13 @@ def list_tag_properties(self, repository, **kwargs):
 
         :param str repository: Name of the repository
         :keyword order_by: Query parameter for ordering by time ascending or descending
-        :paramtype order_by: :class:`~azure.containerregistry.TagOrder` or str
+        :paramtype order_by: ~azure.containerregistry.ArtifactTagOrder or str
         :keyword results_per_page: Number of repositories to return per page
         :paramtype results_per_page: int
-        :returns: An iterable of :class:`~azure.containerregistry.ArtifactTagProperties`
+        :returns: An iterable of ~azure.containerregistry.ArtifactTagProperties
         :rtype: ~azure.core.paging.ItemPaged[~azure.containerregistry.ArtifactTagProperties]
-        :rtype: :class:`~azure.core.paging.ItemPaged`
-        :raises: :class:`~azure.core.exceptions.ResourceNotFoundError`
+        :rtype: ~azure.core.paging.ItemPaged
```

Comment:
This needs a subtype: `:rtype: ~azure.core.paging.ItemPaged[~azure.containerregistry.ArtifactTagProperties]`